libr/main/rasm2.c

```diff
@@ -448,7 +450,9 @@ static int rasm_asm(RAsmState *as, const char *buf, ut64 offset, ut64 len, int b
 					}
 					printf ("\n");
 				} else {
-					print_buf (as, acode->buf_hex);
+					char* str = r_asm_code_get_hex (acode);
+					print_buf (as, str);
```

Comment:
check for null

Reply:
no need. checked inside `print_buf`

Reply:
static functions should have no null pointer checks for the arguments coming from the caller. tahts the rule we decided to use